scripts/index.js

```diff
@@ -0,0 +1,105 @@
+let temp = 75;
+
+const updateTempColor = (tempNum) => {
+    const tempVal = document.getElementById("temp");
+    if (tempNum >= 80) {
+        color = "red";
+    } else if (tempNum >= 70) {
+        color = "orange";
+    } else if (tempNum >= 60) {
+        color = "yellow";
+    } else if (tempNum >= 50) {
+        color = "green";
+    } else{
+        color = "blue";
+    }
+    tempVal.classList = color;
+}
+
+const updateLandscape = (tempNum) => {
+    const landscapeView = document.getElementById("landscape");
+    if (tempNum >= 80) {
+        landscapeView.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    } else if (tempNum >= 70) {
+        landscapeView.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (tempNum >= 60) {
+        landscapeView.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (tempNum >= 50) {
+        landscapeView.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else{
+        landscapeView.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
+}
```

Comment:
Great implementation. You could also include this functionality in the update temp color function because they are both dependable on tempNum.